bears/python/PyUnusedCodeBear.py

```diff
@@ -22,11 +22,12 @@ def run(self, filename, file,
 
         - Unneeded pass statements.
         - Unneeded builtin imports.
+        - Unneeded variables.
 
         :param remove_all_unused_imports:
             True removes all unused imports - might have side effects
         :param remove_unused_variables:
-            True removes unused variables - might have side effects
+            False removes unused variables - might have side effects
```

Comment:
`False` **doesn't** remove unused variables! ;) And maybe exchange *unused* with *the unneeded*, to get some consistency with the list above

And you should move `might have side effects` to the `- Unneeded variables.` list entry above